cmd/cinder-csi-plugin/main.go

```diff
@@ -67,6 +68,7 @@ func main() {
 	}
 
 	cmd.PersistentFlags().StringSliceVar(&cloudNames, "cloud-name", []string{""}, "CSI driver cloud name in config files. This option can be given multiple times to manage multiple openstack clouds")
+	cmd.PersistentFlags().StringToStringVar(&additionalTopologies, "additionnal-topology", map[string]string{}, "CSI driver topology example topology.kubernetes.io/region=REGION1. This option can be given multiple times to manage multiple topology keys")
```

Comment:
Let's rephrase this a bit:

> Additional CSI driver topology keys, for example topology.kubernetes.io/region=REGION1. This option can be specified multiple times to add multiple additional topology keys.

Also there's a typo in the option name, please fix.

Reply:
c98dfe4cbbd70dce0ab0260180ed91c35edb3aec